package.json

```diff
@@ -32,11 +32,14 @@
     "start": "start-storybook -p 9001 -c storybook -s ./storybook/static",
     "build": "lerna run --parallel build -- -- --env production",
     "build:storybook": "build-storybook -c storybook -o ./storybook/public -s ./storybook/static",
+    "setup": "git config core.hooksPath .githooks && npm run init",
     "init": "npm install && lerna bootstrap && npm run build",
     "link-all-packages": "lerna exec --ignore @leafygreen-ui/lib --ignore @leafygreen-ui/theme -- npm link ",
     "link": "./scripts/link.sh",
-    "lint": "eslint \"packages/**/*.js\"",
+    "lint": "npm run prettier:lint && npm run eslint",
+    "eslint": "eslint \"packages/**/*.js\"",
     "prettier": "prettier --write \"**/*.js\"",
+    "prettier:lint": "prettylint \"**/*.{js,ts}\" --config ./.prettierrc",
```

Comment:
[Prettier has this built in](https://prettier.io/docs/en/cli.html#list-different).

Reply:
That's what I thought too, but prettier kept telling me that it didn't recognize `--check`. Spent a bit of time on this yesterday. `prettier --help` doesn't list it as an option either.

Reply:
I think it may be due to our version of Prettier. `--check-different` works and it's what I'd recommend we use too so we can see what files aren't formatted correctly. 

```
prettier --list-different "**/*.{js,ts,tsx}"
```

Reply:
As of this PR we should be on the latest. I don't think `--check-different` returns status code `1` on any mismatch like `prettylint` does.

Good point on adding `tsx`.

Reply:
As far as I'm aware, I don't think that `--check-different` is an option, is it?

Reply:
It doesn't return 0 so pretty sure it works. We use it on Cloud and it's been fine.